tests/integration/call_run_within_script_with_autokeras.py

```diff
@@ -0,0 +1,33 @@
+"""
+Search for a good model for the
+[MNIST](https://keras.io/datasets/#mnist-database-of-handwritten-digits) dataset.
+"""
+
+from tensorflow.keras.datasets import mnist
+import tensorflow_cloud as tfc
+
+import autokeras as ak
+
+# Automated MirroredStrategy: chief config with multiple GPUs
+tfc.run(
+    entry_point=None,
```

Comment:
Can we remove entry_point, distribution_strategy and worker_count params as they are default values?